testutil/integration/router.go

```diff
@@ -53,8 +53,9 @@ func NewIntegrationApp(
 	interfaceRegistry := codectypes.NewInterfaceRegistry()
 	moduleManager := module.NewManagerFromMap(modules)
 	moduleManager.RegisterInterfaces(interfaceRegistry)
+	signingCtx := interfaceRegistry.SigningContext()
```

Comment:
This one will be nil, we should take the address codecs as arguments of `NewIntegrationApp`